app/javascript/components/pages/LastCallShow.js

```diff
@@ -1,74 +1,59 @@
-import React from 'react'
-import { useState } from 'react'
-import { Card, Button } from 'react-bootstrap'
-import { NavLink, useParams, Navigate } from 'react-router-dom'
+import React, { useState } from "react";
+import { Card, Button } from "react-bootstrap";
+import { NavLink, useParams, Navigate } from "react-router-dom";
 
 export default function LastCallShow(props) {
-  let [remove, setRemove] = useState(false)
-  
-  let { logged_in, deleteCocktail, readCocktails} = props
-  let { id } = useParams()
-  
-  let { cocktails } = props
-  let cocktail = cocktails.find(cocktailObj => cocktailObj.id == id)
-  console.log(cocktail)
+  let [remove, setRemove] = useState(false);
 
-  const handleDelete = (id) => {
-    deleteCocktail(id)
-    readCocktails()
-    setRemove(true)
-  }
+  let { logged_in, deleteCocktail, readCocktails } = props;
+  let { id } = useParams();
+
+  let { cocktails } = props;
```

Comment:
use const instead of let for line 8, 9, 11

since, they are immutables, read-only